lib/rodauth/rails/feature.rb

```diff
@@ -28,6 +28,24 @@ def render(page)
         super
     end
 
+    if Rodauth::MAJOR >= 2 && Rodauth::MINOR >= 1
+      # Verify Rails' authenticity token.
+      def check_csrf
+        rails_check_csrf!
+      end
+
+      # Have Rodauth call #check_csrf automatically.
+      def check_csrf?
+        true
```

Comment:
I guess this can be true because rails will always fallback on its own config to decide if csrf must really be checked, right?

Reply:
That’s correct 👍